cmd/agent/install_script.sh

```diff
@@ -322,6 +323,14 @@ else
   $sudo_cmd chmod 640 $CONF
 fi
 
+# Creating or overriding the install information
+echo "---
+install_method:
+  tool: install_script
+  tool_version: install_script
+  installer_version: $install_script_version
```

Comment:
```suggestion
  installer_version: installer_version-$install_script_version
```
For consistency?

Reply:
Definitely!

Reply:
I made a typo but I've seen you fixed it :)